clients/venice-admin-tool/src/main/java/com/linkedin/venice/Arg.java

```diff
@@ -219,7 +219,8 @@ public enum Arg {
       "Type of system store to backfill. Supported types are davinci_push_status_store and meta_store"
   ), RETRY("retry", "r", false, "Retry this operation"),
   DISABLE_LOG("disable-log", "dl", false, "Disable logs from internal classes. Only print command output on console"),
-  STORAGE_PERSONA("storage-persona", "sp", true, "Name of Storage Persona");
+  STORAGE_PERSONA("storage-persona", "sp", true, "Name of Storage Persona"),
+  STORE_VIEW_CONFIGS("storage-view-configs", "svc", true, "Config that describes views to be added for a store");
```

Comment:
We might need to add some example here to inform users to know how to setup these map-like strings, as it is not that straight-forward.

Reply:
done.